README.md

```diff
@@ -55,13 +62,40 @@ To know more about webhint, how to configure it, etc. see the online
 [user guide][user guide], or the [local version][local user guide]
 for the most recent content.
 
+
 ## Contributing to webhint
 
+This project follows a monorepo pattern. That means that the code
+for all the webhint flavors (CLI, browser and VS Code extension,
+hints, formatters, etc.) are in here and are published as separate
+npm packages.
+
+To build the project from the source you will need to install
+a recent version of node and [yarn][]. Once you've done this run
+the following from the root of your cloned version:
+
+```bash
+yarn
+yarn build
+```
+
+This can take a bit so please be patient.
+
 To know more about the internals of `webhint`, the structure of the
 project, how to create new hints, collectors, formatters, etc, take
```

Comment:
Hmm... was "collectors" an older term in the project? Or maybe it was meant to be "connectors". Not sure, but we should fix it while we're here.
```suggestion
project, how to create new hints, parsers, formatters, etc, take
```